config/config.js

```diff
@@ -0,0 +1,11 @@
+const Sequelize = require("sequelize");
```

Comment:
remove Sequelize @mukeshsin

Reply:
ok sir @meanstack9716  done

Reply:
can i merge it start next
@meanstack9716